package.json

```diff
@@ -13,7 +13,13 @@
   },
   "scripts": {
     "test": "mocha --require @babel/register",
-    "build": "node build.js"
+    "build": "node build.js",
+    "build-addressGenerator": "cd example/addressGenerator/ && node build.js",
+    "build-keychain": "cd example/keychain/ && node build.js",
+    "build-publicKeyGenerator": "cd example/publicKeyGenerator/ && node build.js",
+    "build-transactionBuilder": "cd example/transactionBuilder/ && node build.js",
+    "build-examples": "npm run build-addressGenerator && npm run build-keychain && npm run build-publicKeyGenerator && npm run build-transactionBuilder",
```

Comment:
For the `build-example` command, you can use something like this : 
`"build-examples": "find ./example -maxdepth 2 -name package.json -execdir npm run build \\;"`
This will remove the need to create a build command for all example

Reply:
+1